src/auth/auth-config.ts

```diff
@@ -1750,6 +1750,21 @@ export interface RecaptchaManagedRule {
  action?: RecaptchaAction;
 }
 
+/**
+ * The managed rules for toll fraud provider, containing the enforcement status. 
+ * The toll fraud provider contains all SMS related user flows.
+ */
+export interface RecaptchaTollFraudManagedRule {
+  /**
+  * The action will be enforced if the reCAPTCHA score of a request is larger than endScore.
```

Comment:
Should it be `startScore` instead of `endScore` in the doc? Also should we mention the range of values that is allowed and add that into our validation logic?